### PR TITLE
[WIP] example proposal.json files for param changes

### PR DESCRIPTION
**Pull request by denalimarsh**
Each proposal.json file is co-located with an associated genesis.json file which describes the initial state. Proposal deposit/voting time is set to 5 minutes.

### CODE REVIEW

File: contrib/proposal_examples/addcollateral/addcollateral2.json [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+      ]
+    },
+    {
+      "subspace": "cdp",
+      "key": "DebtParams",
+      "value": [
+          {
+              "conversion_factor": "6",
+              "debt_floor": "10000000",
+              "debt_limit": [
```
Comment (karzak) [inline contrib/proposal_examples/addcollateral/addcollateral2.json]:
```
              "debt_limit": [
                {
                  "amount": "3000000000000",
                  "denom": "usdx"
                }
              ],
```

Remove. DebtParam doesn't have `debt_limit` field anymore.

File: contrib/proposal_examples/addcollateral/addcollateral2.json [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+          }
+      ]
+    },
+    {
+      "subspace": "cdp",
+      "key": "GlobalDebtLimit",
+      "value": [
+          {
+              "global_debt_limit": [
+                  {
```
Comment (karzak) [inline contrib/proposal_examples/addcollateral/addcollateral2.json]: The value should just be an `sdk.Coins`:

```
"value":  [
                  {
                  "amount": "3000000000000",
                  "denom": "usdx"
                  }
              ]
```

**Comment by denalimarsh** [comment]
Implemented requested changes and tested. Bunch of duplicate commits from force pushing the branch though. 

There's 1-2 other cli txs which surpass the 200,000 gas limit as well, I've been using 500,000 on every cli tx to sidestep the issue. Is there any reason why we shouldn't increase the `max_bytes` and/or `max_gas` in genesis.config?

File: contrib/proposal_examples/addcollateral/addcollateral2.json (hunk 2 above)
Comment (denalimarsh) [reply contrib/proposal_examples/addcollateral/addcollateral2.json]: https://github.com/Kava-Labs/kava/pull/331/commits/2130feff6fabfb6a510cc26dde0208ac0b8531a0

File: contrib/proposal_examples/addcollateral/addcollateral2.json (hunk 1 above)
Comment (denalimarsh) [reply contrib/proposal_examples/addcollateral/addcollateral2.json]: https://github.com/Kava-Labs/kava/pull/331/commits/2130feff6fabfb6a510cc26dde0208ac0b8531a0

**Comment by karzak** [comment]
`max_gas`and `max_bytes` just control the per block limits. I wouldn't feel comfortable altering them due to potential issues with latency in gossiping larger blocks. The default gas estimate is set in https://github.com/cosmos/cosmos-sdk/blob/master/client/flags/flags.go and is not configurable. I'll see if anyone has opened as issue regarding that, as it should probably be configurable per-client.

**Pull request merged**